browser/brave_shields/brave_shields_web_contents_observer.cc

```diff
@@ -212,6 +218,15 @@ void BraveShieldsWebContentsObserver::DispatchBlockedEventForWebContents(
     event_router->BroadcastEvent(std::move(event));
   }
 #endif
+  if (base::FeatureList::IsEnabled(
+          brave_shields::features::kBraveShieldsPanelV2)) {
+#if !defined(OS_ANDROID)
+    if (!web_contents)
+      return;
+    brave_shields::BraveShieldsDataController::FromWebContents(web_contents)
```

Comment:
you need to check to see if the BraveShieldsDataController is null

Reply:
this is bubble's web content

Reply:
adding a DCHECK here